generic/tests/cfg/linux_stress.cfg

```diff
@@ -1,6 +1,19 @@
 - linux_stress:
     only Linux
     type = linux_stress
-    stress_duration = 60
-    # Please specify below params for stress test, or use default value as shown
-    # download_url = http://people.seas.harvard.edu/~apw/stress/stress-1.0.4.tar.gz
```

Comment:
Can we keep the original test app `stress-1.0.4.tar.gz` as `- @default:` and below `stressapptest` as another variants `- sressapptest:`? Cause we might need the original one for some of our test cases.

Reply:
:+1:  done.

Reply:
done